estimation_tracker/tracker/migrations/0002_remove_estimate_time_calc.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.0.4 on 2022-05-02 11:32
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Optional: Could you just recreate a unique migration by excluding the field `estimated_time_by_calc `?